packages/epo-react-lib/src/form/HorizontalSlider/HorizontalSlider.tsx

```diff
@@ -0,0 +1,98 @@
+import { useState, FunctionComponent } from "react";
+import * as Styled from "./styles";
+
+type SliderValue = number | readonly number[];
+
+interface HorizontalSliderProps {
+  min?: number;
+  max?: number;
+  step?: number;
+  value: number | number[];
+  label: string;
+  minLabel?: string;
+  maxLabel?: string;
+  labelledbyId?: string;
+  onChangeCallback: (value: number | number[], label: string) => void;
+  color?: string;
+  darkMode?: boolean;
+}
+
+const HorizontalSlider: FunctionComponent<HorizontalSliderProps> = ({
+  min = 0,
+  max = 100,
+  step = 1,
+  value,
+  onChangeCallback,
+  label,
+  minLabel,
+  maxLabel,
+  labelledbyId,
+  color = "transparent",
+  darkMode = false,
+}) => {
+  const [showThumbLabels, setShowThumbLabels] = useState(false);
+  const hasDoubleHandles = Array.isArray(value) && value.length > 1;
+
+  const handleChange = (value: SliderValue) => {
+    setShowThumbLabels(false);
+    if (onChangeCallback) onChangeCallback(value as number | number[], label);
+  };
+
+  const Track = (props: any, state: { index: number; value: SliderValue }) => {
+    const { index } = state;
+    const hasColor =
+      (hasDoubleHandles && index === 1) || (!hasDoubleHandles && index === 0);
+
+    return <Styled.Track color={hasColor ? color : "transparent"} {...props} />;
```

Comment:
adds a user defined color to either the left-most track for single value sliders, or the second track for double value sliders.